libs/blocks/merch/merch.js

```diff
@@ -302,11 +302,20 @@ async function openFragmentModal(path, getModal) {
   return modal;
 }
 
+function focusOnTab(url) {
+  const metaPreselectPlan = document.querySelector('meta[name="preselect-plan"]');
+  if (!metaPreselectPlan) return url;
+  const urlWithPlan = new URL(url);
+  urlWithPlan.searchParams.set('plan', metaPreselectPlan.content);
```

Comment:
Should `metaPreselectPlan.content` be checked as well? If it has no value, then the URL will be `...?plan=`

Reply:
Right. The check added
`if (!metaPreselectPlan || !metaPreselectPlan.content) return url;`